doc/projects.rst

```diff
@@ -2,29 +2,30 @@
+With the :ref:`Project <project-explorer>` pane, you can browse all your project's files, regardless of your current working directory or :doc:`fileexplorer` location.
 
-* Opening, closing or switching to a :guilabel:`Project` automatically saves and restores your :doc:`editor` panes and open files to exactly how you left off.
-  This allows you to easily switch between many different development tasks without having to manually re-create your session for each one.
-* Your project's root directory is automatically added to the ``PYTHONPATH``, so you can easily ``import`` and work with any modules and packages you create with zero setup.
-* The project path is also used to automatically set your working directory, and can be used as an automatic preset for several modules, such as the :doc:`findinfiles` search location.
-* You can browse all your :guilabel:`Project` files from the :ref:`Project Explorer <project-explorer>`, regardless of your current working directory or :doc:`fileexplorer` location.
-* :guilabel:`Projects` are :ref:`integrated <vcs-section>` with the ``git`` version control system, allowing you to commit files and open them or your repository in the ``gitk`` GUI right from within Spyder.
+.. image:: images/projects/projects-main.png
+   :alt: Spyder showing Project Explorer and projects menu
+
+
+In addition, your project's root folder is used to set your working directory, and automatically added to the ``PYTHONPATH``, so you can easily ``import`` and work with any modules and packages you create inside of it.
 
 .. note::
 
-   :guilabel:`Projects` are completely optional and not imposed on users.
-   All of Spyder' functionality (session saving, File Explorer, working directory, etc) is available without creating a :guilabel:`Project`, just on a global rather than :guilabel:`Project`-specific basis.
+   Projects are completely optional and not imposed on users.
+   All of Spyder' functionality (code completion, session saving, File Explorer, working directory, etc) is available without creating a :guilabel:`Project`.
+
 
 
 ==================
 Creating a Project
 ==================
 
-To create a :guilabel:`Project`, click the :guilabel:`New Project` entry in the :guilabel:`Projects` menu, choose whether you'd like to associate a :guilabel:`Project` with an existing directory or make a new one, and enter the :guilabel:`Project` 's name and path:
+To create a :guilabel:`Project`, click the :guilabel:`New Project` entry in the :guilabel:`Projects` menu, choose whether you'd like to associate a :guilabel:`Project` with an existing directory or make a new one, and enter the :guilabel:`Project`'s name and path.
 
-.. image:: images/projects/projects-new-project.png
-   :alt: New project dialog, with options to set the name, type and location
+.. image:: images/projects/projects-new.gif
```

Comment:
I suggest creating a project in an existing directory, so that something is actually shown in the projects pane.

Reply:
There is a screenshot above with content on it, but as a new user probably you wouldn’t have any content. Let’s see what @ccordoba12 thinks

Reply:
* I agree with @CAM-Gerlach. You usually want to set a project for an existing directory.
* Also, @juanis2112 please make the Projects pane wider before creating the new project. That way the pane won't appear with such an small width in the gif.
* By the way, not setting a minimal width for the pane is a bug that we need to solve. So, could you open an issue about it so we don't forget to do it for 4.2.2? Thanks!

Reply:
Agree with all; was going to mention it was rather narrow but didn't know about changing the (overly narrow) default.

Reply:
Looks like to me that everything here is resolved and the GIF looks great, so this should be good to go.

Reply:
Agreed.